### DISCUSSION TITLE
账号密码问题

### DISCUSSION THREAD

**Discussion by AvavavaDina** [Melledy/Grasscutter#2491] [General]
在输入密码时我是随机输入的，因此我忘记了我的密码，在MongoDB软件中我只能找到被加密的sessionKey，我应该如何找回我的账号密码

**Comment by RoundNailoong** [comment]
默认情况下所有账号都是不存储密码的 所以再输入一个新密码同样也可以登录

**Reply by AvavavaDina** [reply]
感谢
